frontend/apps/pwa/src/features/former/FieldSections.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useState } from 'react';
+
+import { FormField } from '../../reducers/Former/types';
+
+import { FieldTypePicker } from './FieldTypePicker';
+import { FormerField } from './FormerField';
+import { FormerProps } from './types';
+
+type FieldSectionsProps = Omit<
```

Comment:
If props are contained to the file I think `Props` suffices as a type name. It's useful to name the props if they're exported and used elsewhere.

Reply:
I don't think it hurts to give local types a more specific name, but fine